aiogram_dialog/manager/manager.py

```diff
@@ -50,9 +51,16 @@ def storage(self) -> StorageProxy:
         return self.data[STORAGE_KEY]
 
     async def _remove_kbd(self) -> None:
-        chat = get_chat(self.event)
-        message = Message(chat=chat, message_id=self.current_stack().last_message_id)
-        await remove_kbd(self.event.bot, message)
+        chat = self.data['event_chat']
+        bot = self.data['bot']
+
+        message = None
+        if self.current_stack().last_message_id:
+            message = Message(chat=chat,
+                              message_id=self.current_stack().last_message_id,
+                              date=datetime.now()  # ToDo: check this
+                              )
+        await remove_kbd(bot, message)
```

Comment:
indent?

Reply:
pydantic warning, maybe create new class for these cases?

```
pydantic.error_wrappers.ValidationError: 1 validation error for Message
date
  field required (type=value_error.missing)
```